<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-06-18 09:37:12.247 ***</b>
=ERROR REPORT==== 18-Jun-2024::09:37:12.247272 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244049.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1973}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1934}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-06-18 09:37:12.248 ***</b>
=CRASH REPORT==== 18-Jun-2024::09:37:12.247711 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244054.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2426)
    ancestors: [<0.244049.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244049.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6886
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-06-18 09:37:52.583 ***</b>
=ERROR REPORT==== 18-Jun-2024::09:37:52.583732 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248400.0&gt;,
                               {#Ref&lt;0.1359947205.2701393921.222921&gt;,1008158,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.1359947205.2701393921.222921&gt;,1008158,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-06-18 09:37:52.584 ***</b>
=CRASH REPORT==== 18-Jun-2024::09:37:52.584131 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248420.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.1359947205.2701393921.222921&gt;,1008158,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [&lt;0.248400.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 4750
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-06-18 09:42:31.391 ***</b>
=CRASH REPORT==== 18-Jun-2024::09:42:31.390835 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254159.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.1359947205.2702704641.228807&gt;,5626,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1694)
    ancestors: [&lt;0.254157.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28123
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-06-18 09:42:41.529 ***</b>
=ERROR REPORT==== 18-Jun-2024::09:42:41.529417 ===
** Generic server <0.255443.0> terminating 
** Last message in was {nodedown,'peer_down_crash-534050-11826@f78c8d9bc80e'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-534050-11826",
                              peer_down => crash},
                            'peer_down_crash-534050-11826@f78c8d9bc80e',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,48,47,98,105,110,
                             47,101,114,108],
                            ["-sname","peer_down_crash-534050-11826",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQGY3OGM4ZDliYzgwZQAD5dMAAAAAZnE2wQ=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-534050-11826@f78c8d9bc80e'}},
                            {<0.255434.0>,
                             #Ref<0.1359947205.2702704641.242931>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-534050-11826@f78c8d9bc80e'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-06-18 09:42:41.530 ***</b>
=CRASH REPORT==== 18-Jun-2024::09:42:41.530087 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255443.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-534050-11826@f78c8d9bc80e'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [<0.255434.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11353
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-06-18 09:42:42.258 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:42:41.256434 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-425445-11826@f78c8d9bc80e' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-06-18 09:42:42.642 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:42:42.642106 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-425509-11826@f78c8d9bc80e'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-06-18 09:42:42.784 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:42:42.784192 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-368900-11826@f78c8d9bc80e'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-06-18 09:42:58.868 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:42:57.866292 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-369380-11826@f78c8d9bc80e' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-06-18 09:42:59.249 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:42:59.249331 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-425571-11826@f78c8d9bc80e'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-06-18 09:42:59.353 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:42:59.353281 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-425539-11826@f78c8d9bc80e'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-18 09:43:01.515 ***</b>
=ERROR REPORT==== 18-Jun-2024::09:43:01.514963 ===
** Generic server <0.255796.0> terminating 
** Last message in was {'EXIT',<0.255768.0>,
                               {#Ref<0.1359947205.2695888898.11216>,1559075,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-425861-11826",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-425861-11826@f78c8d9bc80e',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,48,47,98,105,110,47,101,114,108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-425861-11826",
                                     "-user","peer"],
                                    #Port<0.3118>,undefined,<<>>,running,
                                    {<0.255768.0>,
                                     #Ref<0.1359947205.2695626756.80409>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.1359947205.2695888898.11216>,1559075,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-18 09:43:01.515 ***</b>
=CRASH REPORT==== 18-Jun-2024::09:43:01.515534 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255796.0>
    registered_name: []
    exception exit: {#Ref<0.1359947205.2695888898.11216>,1559075,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.255768.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3118>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 13579
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-18 09:43:02.133 ***</b>
=ERROR REPORT==== 18-Jun-2024::09:43:02.132935 ===
** Generic server <0.255830.0> terminating 
** Last message in was {'EXIT',<0.255774.0>,
                           {#Ref<0.1359947205.2695888899.240651>,2175438,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "DRUXBOFMNAKDSEQLLHBF","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@f78c8d9bc80e.2024-06-18_09.26.56/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-425859-11826",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@f78c8d9bc80e.2024-06-18_09.26.56/make_test_dir.stdlib_test.logs/run.2024-06-18_09.27.43/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-425859-11826",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-425859-11826@f78c8d9bc80e',
                                    "/usr/local/lib/erlang-26.2.5/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-425859-11826",
                                     "-setcookie","DRUXBOFMNAKDSEQLLHBF",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@f78c8d9bc80e.2024-06-18_09.26.56/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-425859-11826",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@f78c8d9bc80e.2024-06-18_09.26.56/make_test_dir.stdlib_test.logs/run.2024-06-18_09.27.43/log_private/",
                                     "-user","peer"],
                                    #Port<0.3145>,undefined,<<>>,running,
                                    {<0.255774.0>,
                                     #Ref<0.1359947205.2695888898.13450>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.1359947205.2695888899.240651>,2175438,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-18 09:43:02.133 ***</b>
=CRASH REPORT==== 18-Jun-2024::09:43:02.133708 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255830.0>
    registered_name: []
    exception exit: {#Ref<0.1359947205.2695888899.240651>,2175438,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.255774.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3145>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 17731
    stack_size: 29
    reductions: 22271
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-18 09:43:02.526 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:43:01.524992 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-425827-11826@f78c8d9bc80e' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-18 09:43:02.578 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:43:02.578301 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-369700-11826@f78c8d9bc80e'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-18 09:43:02.925 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:43:02.925288 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-369828-11826@f78c8d9bc80e'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-18 09:43:02.947 ***</b>
=WARNING REPORT==== 18-Jun-2024::09:43:02.947185 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-369796-11826@f78c8d9bc80e'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-06-18 09:53:48.155 ***</b>
=ERROR REPORT==== 18-Jun-2024::09:53:48.155586 ===
** Generic server <0.302516.0> terminating 
** Last message in was {'EXIT',<0.302513.0>,
                               {#Ref<0.1359947205.2711879682.60646>,350841,
                                [{child1,<0.302522.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302516.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302522.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459141],
                            0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.1359947205.2711879682.60646>,350841,
    [{child1,<0.302522.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-06-18 09:53:48.156 ***</b>
=ERROR REPORT==== 18-Jun-2024::09:53:48.155586 ===
** Generic server <0.302515.0> terminating 
** Last message in was {'EXIT',<0.302513.0>,
                               {#Ref<0.1359947205.2711879682.60646>,350841,
                                [{child1,<0.302522.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302515.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302521.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459141],
                            0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.1359947205.2711879682.60646>,350841,
    [{child1,<0.302522.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-06-18 09:53:48.156 ***</b>
=CRASH REPORT==== 18-Jun-2024::09:53:48.156044 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302516.0>
    registered_name: []
    exception exit: {#Ref<0.1359947205.2711879682.60646>,350841,
                     [{child1,<0.302522.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.302513.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11943
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-06-18 09:53:48.156 ***</b>
=CRASH REPORT==== 18-Jun-2024::09:53:48.156321 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302515.0>
    registered_name: []
    exception exit: {#Ref<0.1359947205.2711879682.60646>,350841,
                     [{child1,<0.302522.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.302513.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11914
  neighbours:


</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Tue Jun 18 2024 09:27:43<!--/date--><br />
</div>
</center>
</body>
</html>
